src/ERC20Extended.sol

```diff
@@ -128,7 +128,7 @@ abstract contract ERC20Extended is IERC20Extended, IERC5267, ERC3009 {
     \******************************************************************************************************************/
 
     /// @inheritdoc IERC20
-    function balanceOf(address account_) external view virtual returns (uint256) {}
+    function balanceOf(address account_) external view virtual returns (uint256);
```

Comment:
This line will be deleted anyway since there is n point for abstract contracts to have unused virtual functions if they are already defined in the interface.